### PR TITLE
Update Ingestion API get current user docs

### PR DESCRIPTION
**Pull request by noralin**
https://greenhouseio.atlassian.net/browse/GREEN-28712

related GHR card https://greenhouseio.atlassian.net/browse/GREEN-28505

### CODE REVIEW

**Comment by noralin** [comment]
image

**Review by capablemonkey — APPROVED** [review]
Nice work!

File: source/includes/candidate-ingestion/_current_user.md [hunk 1]
```diff
@@ -28,8 +30,10 @@ This request is simply the URL. It takes no parameters.
 
 ### Response Parameters
 
-Property Name | Type | Required
--------------- | -------------- | -------------- 
-first_name | String | Yes
-last_name | String | Yes
-email | String | Yes
\ No newline at end of file
+Property Name | Type | Required | Description
```
Comment (glovguy) [inline source/includes/candidate-ingestion/_current_user.md:33]: Is `required` a meaningful column on a read-only endpoint?

Comment (noralin) [reply source/includes/candidate-ingestion/_current_user.md:33]: that's a good point! i shall remove it

Comment (noralin) [reply source/includes/candidate-ingestion/_current_user.md:33]: it's funny it's listed in so many other places for Get request, should i remove them all?

**Review by glovguy — APPROVED** [review]

**Pull request merged**